workflow/controller/operator.go

```diff
@@ -2190,7 +2190,7 @@ func (woc *wfOperationCtx) executeContainer(ctx context.Context, nodeName string
 		return node, err
 	}
 
-	woc.log.Debugf("Executing node %s with container template: %v\n", nodeName, tmpl)
+	woc.log.Debugf("Executing node %s with container template: %v\n", nodeName, tmpl.Name)
```

Comment:
this made the debug logs noisy and I have never found it useful in the last 1 year